lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/Plugins.js

```diff
@@ -16,6 +16,9 @@ qx.Class.define("callbackery.ui.Plugins", {
 
         plugins : {
             init: {
+                action: function(pluginConfig) {
+                    return new callbackery.ui.plugin.ActionForm(pluginConfig);
```

Comment:
warum die parent form data nicht auch zur verfuegung stellen ?

Reply:
Es hat kein form ...

Reply:
die actions kriegen die info nicht, wenn man die anklickt ? ich dachte bei den popup forms waere das so

Reply:
Ja, du hast recht. Ich habe es in meiner Anwendung nicht benötigt, aber es ist sicherlich sinnvoll, das bereit zu stellen. Hab's ergänzt.
**ALLERDINGS: ** Diese Erweiterung sollte garnicht Teil dieses PRs sein, daher habe ich diese Änderung erstmal ganz raus genommen und werde sie dann im nächsten PR für's neue MailToken Plugin bringen.